lib/features/search-pad/SearchPadProvider.ts

```diff
@@ -1,10 +1,17 @@
 import type { Element } from '../../model/Types';
 
-export type Token = {
+type LegacyToken = {
   matched?: string;
   normal?: string;
 };
 
+type ModernToken = {
```

Comment:
NABD, but maybe calling this just `Token` is fine? as moving forward this is going to be our _default_ token type right?

Reply:
Nah, I did not go so far to drop `Legacy` tokens, but rather there is now two ways to provide search results.

In the past you provided:

```
[ { normal: 'My ' }, { matched: 'name' }, ... ]
```

The modern way, that allows to use `search` results without conversion is:

```
[ { value: 'My ' }, { value: 'name', match: true }, ... ]
```

We could of course decide to make this a full fledged major breaking change.

I regard the name as fine, because `Token` is what is being exported. `Modern` vs. `Legacy` is really an implementation detail.